docker_daemon/check.py

```diff
@@ -472,17 +472,28 @@ def _extract_tag_value(self, entity, tag_name):
 
         return entity["_tag_values"][tag_name]
 
+    def _localhost_ecs_agent(self):
```

Comment:
I think the method should also be commented or renamed. What we're checking here is if the ecs-agent is local from the pov of dd-agent, network wise. So it could be renamed `_is_ecs_agent_local` or commented with something like `return True if we can reach the ecs-agent over localhost, False otherwise. This is needed because if the ecs-agent is started with --net=host it won't have an IP address attached`.